src/index.html

```diff
@@ -50,7 +50,7 @@ <h1 class="profile__title"></h1>
     <!-- EDIT AVATAR IMAGE -->
     <div class="modal" id="edit-avatar-modal">
       <div class="modal__container modal__container_avatar">
-        <button class="modal__close" id="avatar-close-button"></button>
+        <button class="modal__close-button" type="button" ></button>
```

Comment:
` class="modal__close-button"` you don't have such styles for the close icon. Please, be more attantive. You have `class="modal__close"` in each close icon. So, it means you need the same styles for the avatar close icon

Reply:
` class="modal__close-button"` you don't have such styles for the close icon. Please, be more attentive. You have `class="modal__close"` in each close icon. So, it means you need the same styles for the avatar close icon